src/app/story/storyrow/storyrow.component.html

```diff
@@ -3,13 +3,24 @@
   <div id="{{ 'js-' + node.resource_type.toLowerCase() + '-node' }}" class="node-column__node" [ngClass]="{'node-column__node--active': active}" 
       [routerLink]="['/', node.resource_type.toLowerCase(), getNodeUid()]"
       tooltip="{{ node.resource_type | nodeTypeDisplay }}: {{ node | nodeUidDisplay }}">
-    <img class="node-column__node-img" src="assets/circle_single.svg"/>
-    <div class="node-column__node-icon"><i class="fa fa-th {{ getNodeIconClass() }}" aria-hidden="true"></i></div>
+    <img class="node-column__node-img" src="assets/circle_single.svg" />
+    <div *ngIf="useSVG();else usePatternfly" class="node-column__node-icon">
+      <svg-icon src="assets/{{ getSVGName() }}" [svgStyle]="{ 'width.px':40 }" aria-hidden="true"></svg-icon>
```

Comment:
Can `[svgStyle]="{ 'width.px':40 }"` be replaced with CSS in `storyrow.component.css`?